package.json

```diff
@@ -19,6 +19,10 @@
     "@angular/platform-browser": "~7.2.0",
     "@angular/platform-browser-dynamic": "~7.2.0",
     "@angular/router": "~7.2.0",
+    "@fortawesome/angular-fontawesome": "^0.3.0",
```

Comment:
Will angular-fontawesome be supported for the foreseeable future? I see the last commit was 7 months ago is it still supported?

Reply:
> Will angular-fontawesome be supported for the foreseeable future? I see the last publish was 7 months ago.

There is a pull request from 14 days ago, looks like there is plans to support Angular 8

Reply:
No worries, was just checking - even though they are doing commits they have not published for a while. For client projects people would be reluctant to pull in any external lib. Just asking the question so your team starts considering that as well. But go ahead when you are ready.